modules/virt-about-creating-network-bridges.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-creating-network-bridges_{context}"]
+= About creating network bridges
+
+You can attach virtual machines to multiple networks by using Linux bridges.
+You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+
+To attach a virtual machine to an additional network:
+
+. Configure a Linux bridge.
+. Configure a bridge network attachment definition for a namespace in the web console or CLI.
++
+[NOTE]
+====
+The network attachment definition must be in the same namespace as the pod or virtual machine.
+====
+. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
```

Comment:
I think you can simplify this: "Edit the virtual machine configuration to attach it to the bridge network attachment definition." That covers both UI and CLI.

Reply:
Unresolving this conversation because I think it is still valid.

Reply:
I did make this change, but that module doesn't exist anymore ---- it's in the _Attaching a virtual machine to an external network_ assembly.

_Note
The network attachment definition must be in the same namespace as the pod or virtual machine.

. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
** In the web console, create a NIC for a new or existing virtual machine.
** In the CLI, include the network information in the virtual machine configuration._